src/main/java/com/example/bankProjetc/Views/Views.java

```diff
@@ -0,0 +1,19 @@
+package com.example.bankProjetc.Views;
+
+public interface Views {
+	public static interface AccountView {
+		public static interface post {
+		}
+	}
+
+	public static interface CustomerView {
+		public static interface post {
+		}
+	}
+
+	public static interface TransactionView {
+		public static interface post {
+		}
+	}
```

Comment:
create separate files for each interface (Account, Customer, Transaction), try to follow solid principles of smaller interface and separate files